index.js

```diff
@@ -60,4 +60,18 @@ module.exports = robot => {
       }
     }
   }
+
+  // Say hi!
+  const NAME = 'introduction-to-github-apps'
+  robot.on('installation_repositories.added', async context => {
+    const includes = context.payload.repositories_added.some(r => r.name === NAME)
+    if (!includes) return
+
+    return context.github.issues.create({
+      owner: context.payload.installation.account.login,
+      repo: NAME,
+      title: 'Great job installing WIP!',
+      body: 'Some body'
```

Comment:
Are these arcuate?

```
title: 'Great job installing Request Info!',
body: 'Yay!!'
```

Reply:
Nope - @hollenberry needs to fix 'em

Reply:
@JasonEtco can we still just add an issue comment on the open PR?

Reply:
If so, can we set the body to... 

'Well done! You successfully installed the request info app

_disclaimer_ If you use this app in future repos, you won't get a message like this. This is just for Learning Lab!`

Or something similar.

cc: @a-a-ron for feedback

Reply:
Let me know if that works @JasonEtco so we can update this